build-support/src/main/kotlin/app/cash/redwood/buildsupport/RedwoodBuildExtension.kt

```diff
@@ -18,9 +18,6 @@ package app.cash.redwood.buildsupport
 interface RedwoodBuildExtension {
   fun targets(group: TargetGroup)
 
-  /** Add the Compose compiler plugin. */
-  fun composeCompiler()
```

Comment:
The new `org.jetbrains.kotlin.plugin.compose` Gradle plugin now adds the Compose compiler in the same way we were. All callers of this were switched to that.